### PR TITLE
comp string man 1-Bo Mercado

### PR DESCRIPTION
**Pull request by rbomercado**
image

### CODE REVIEW

**Comment by rbomercado** [comment]
image

**Review by codingcodymiller — CHANGES_REQUESTED** [review]
Hey Bo, it looks like you still have one test failing, please get that fixed.

File: string-manipulation-1/src/is-lower-cased.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+if they are all lower case=true if not = false
+retireve true or false
+*/
+function isLowerCased(word) {
+  var words = false;
+  for (var i = 0; i < word.length; i++) {
+    if (word === word.toLowerCase()) {
+      words = true;
+    }
+  }
```
Comment (thebearingedge) [inline string-manipulation-1/src/is-lower-cased.js:15]: Try doing this without the loop. Your `if` statement is already solving the problem.

File: string-manipulation-1/src/is-upper-cased.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+if they are all upper case=true if not=false
+retireve true or false
+*/
+function isUpperCased(word) {
+  var words = false;
+  for (var i = 0; i < word.length; i++) {
+    if (word === word.toUpperCase()) {
+      words = true;
+    }
+  }
```
Comment (thebearingedge) [inline string-manipulation-1/src/is-upper-cased.js:16]: Same here

File: string-manipulation-1/src/is-vowel.js [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+
+function isVowel(char) {
+  var vowel = false;
+  for (var i = 0; i < char.length; i++) {
+    // eslint-disable-next-line eqeqeq
+    if (char === 'e' || char === 'A' || char === 'a' || char === 'E' ||
+    char === 'I' || char === 'i' || char === 'o' || char === 'O' || char === 'u' || char === 'U') {
+      vowel = true;
+    }
+  }
```
Comment (thebearingedge) [inline string-manipulation-1/src/is-vowel.js:18]: If you make the character lowercased first, one time, then you can check half as many comparisons.

```js
var lowerChar = char.toLowerCase();
```

Also please try this without the loop.

**Review by thebearingedge — CHANGES_REQUESTED** [review]

**Comment by rbomercado** [comment]
image

**Review by thebearingedge — APPROVED** [review]

**Pull request merged**